app/Http/Controllers/Admin/AdminController.php

```diff
@@ -16,7 +17,9 @@ public function __construct()
 
     public function index()
     {
-        $users = User::findAllPaginated();
+        $users = ($search = request('search'))
+                ? SearchUsers::get($search)
+                : User::findAllPaginated();
```

Comment:
Replace this with

```php
$users = request()->has('search') ? SearchUsers::get(request('search')) : User:: findAllPaginated();
```

This is because the current implementation will return all the users if a person didn't fill anything into the search box, where the implementation above will always perform the search if the search form was submitted.